frontend/src/lib/model.js

```diff
@@ -247,6 +247,35 @@ async function updateUserName(userId, newName) {
 	}
 }
 
+async function updateStatus(userId, status) {
+	const allUsers = get(users);
+	const user = allUsers[userId];
+
+	try {
+		const response = await fetch(`/api/user/`, {
+			method: 'PATCH',
+			headers: {
+				'Content-Type': 'application/json'
+			},
+			body: JSON.stringify({ name: user.name, status: status })
```

Comment:
You don't need to go out of your way to send the old name along with the new status. Just omit the `name` field.